.asf.yaml

```diff
@@ -35,12 +35,9 @@ github:
     # Enable projects for project management boards
     projects: true
   enabled_merge_buttons:
-    # enable squash button:
     squash:  true
-    # disable merge button:
     merge:   false
-    # disable rebase button:
-    rebase:  false
+    rebase:  true
```

Comment:
It might help to provide committers with documentation on when to use which button. We haven't had the option previously, so this might confuse some. I hope we don't get any committers arguing for a force push update to master to change the history because they hit the wrong button.

Reply:
OK. Let me start a mail thread first.

For documentation, I don't think committers read comments in `.asf.yaml` so probably a page under https://pulsar.apache.org/contribute/develop-coding-conventions/ said "Reviewing Pull Request" or "Merging Pull Request".

Reply:
Thanks @tisonkun. I agree with putting it in that section. A committer guide would be valuable--I don't think we have one though.